Projects/18_Clustering/Predict_Sales.py

```diff
@@ -0,0 +1,36 @@
+# import data for offers
+import pandas as pd
+df_offers = pd.read_excel("./WineKMC.xlsx", sheetname=0)
+df_offers.columns = ["offer_id", "campaign", "varietal", "min_qty", "discount", "origin", "past_peak"]
+
+# import data for transactions
+df_transactions = pd.read_excel("./WineKMC.xlsx", sheetname=1)
+df_transactions.columns = ["customer_name", "offer_id"]
+df_transactions['n'] = 1
+
+# estimate total sales per offer_id
+df_total_sales = pd.merge(df_offers['offer_id'].to_frame(), df_transactions)
```

Comment:
If you want to extract a dataframe (even if it's one column) you can index by a list, for example  `df_offers[['offer_id]]` so you don't need to later turn it into a frame.